src/testRunner/unittests/services/goToDefinition.ts

```diff
@@ -0,0 +1,140 @@
+import { protocol } from "../../_namespaces/ts.server";
+import { baselineTsserverLogs, createLoggerWithInMemoryLogs, createSession } from "../helpers/tsserver";
+import { createServerHost, File } from "../helpers/virtualFileSystemWithWatch";
+
+describe("unittests:: services:: goToDefinition", () => {
```

Comment:
I feel like we should try as much as possible to make sure that things work no matter the order; it may be the case that VS Code does it a certain way but I think it's super likely that some other client will do it that way. I know that in LSP, these orders are totally possible.